src/plugins/ui_stateful/clipboard.ts

```diff
@@ -476,26 +479,25 @@ export class ClipboardPlugin extends UIPlugin {
     return this.clipboardId;
   }
 
-  getClipboardContent(): ClipboardContent {
+  getClipboardContent(): OSClipboardContent {
     return {
       [ClipboardMIMEType.PlainText]: this.getPlainTextContent(),
       [ClipboardMIMEType.Html]: this.getHTMLContent(),
-      [ClipboardMIMEType.OSpreadsheet]: this.getSerializedGridData(),
     };
   }
 
-  private getSerializedGridData(): string {
+  private getgridData(): HtmlClipboardData {
```

Comment:
```suggestion
  private getGridData(): HtmlClipboardData {
```
that's the name we had before so I'm fine if you want to keep it, but why grid ? What does the clipboard plugin knows about the grid ?

Reply:
At first, the clipboard would return a table representing the values inside the sheet "grid". grid is not explicit in this case as it clearly stems from the visual representation, sheetData might be better?